jme3-core/src/main/java/com/jme3/audio/Listener.java

```diff
@@ -130,32 +131,29 @@ public Vector3f getVelocity() {
 
     /**
      * Gets the left direction vector of the listener.
-     * This vector is derived from the listener's rotation.
      *
      * @return The listener's left direction as a {@link Vector3f}.
      */
     public Vector3f getLeft() {
-        return rotation.getRotationColumn(0);
+        return rotation.getRotationColumn(0, left);
```

Comment:
This changes the behaviour. Previously this returned a fresh Vector3f that you had exclusive ownership of. Now you are returned a vector3f that may randomly change at some future point.

Unless this is giving a major boost in performance in typical flows I don't think we should do this

Reply:
This solution maintains consistency with the existing getLocation(), getRotation(), and getVelocity() methods, which already expose internal variables. This approach is not only consistent but also crucial for performance optimization and efficient reuse of internal variables, preventing new vector allocations every time.

The primary reason for this approach is that `Listener` is an internally managed class by `ALAudioRenderer`. It's designed as a unique instance created by `LegacyApplication `and is not intended for external direct use. Therefore, the concerns you've raised regarding its internal exposure are mitigated by its strictly internal scope and established usage patterns.